### PR TITLE
Query multiple tables

### PR DESCRIPTION
**Pull request by inspecta**
- Created the `owner's table`
- Created the `species table`
- Modified the `animals table`
-- Removed species column
-- Added` species_id as a foreign key` references species table
-- Added` owner_id as a foreign key` referencing the owner's table
- Added all the SQL statements into the schema.sql file
- Inserted data into the owners' table
- Inserted data into the species' table
- Modified animals table data to include owner information
- Wrote and executed SQL Queries with SQL JOIN
- Added all queries to the queries.sql file
- Took screenshots of all my results

** Screenshot showing the contents of all the tables**

Screenshot from 2022-12-07 22-36-27

**Screenshots showing results from the JOIN queries **

Screenshot from 2022-12-07 22-33-29

Screenshot from 2022-12-07 22-34-13

### CODE REVIEW

File: data.sql [hunk 1]
```diff
@@ -11,3 +11,26 @@ INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
...
+INSERT INTO owners(full_name, age) VALUES ('Jeniffer Orwell', 19);
+INSERT INTO owners(full_name, age) VALUES ('Bob', 45);
+INSERT INTO owners(full_name, age) VALUES ('Melody Pond', 77);
+INSERT INTO owners(full_name, age) VALUES ('Dean Winchester', 14);
+INSERT INTO owners(full_name, age) VALUES ('Jodie Whittaker', 38);
+
+/* Insert details of animal species */
+INSERT INTO species (name) VALUES ('Pokemon');
+INSERT INTO species (name) VALUES ('Digimon');
+
```
Comment (BenMukebo) [inline data.sql:26]: - [ ] The `animals` table should be modified to meet these requirements, Kindly make sure to add these project requirements 👇🏻 
- if an animal's name ends with `mon` it should have `Digimon` as a `species_id`, all other animals should have `Pokemon` instead here is the screenshot of the project requiement ⬇️ 

Screenshot from 2022-12-07 22-19-00

File: data.sql [hunk 2]
```diff
@@ -11,3 +11,26 @@ INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
...
+/*Update animal's table */
```
Comment (BenMukebo) [inline data.sql]: - [ ] You're missing this project requirement: `Modify your inserted animals to include owner information (owner_id):
Sam Smith owns Agumon.` Kindly address this 👍🏻

File: data.sql [hunk 3]
```diff
@@ -11,3 +11,26 @@ INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
...
+UPDATE animals SET owner_id = 2 WHERE name = 'Gabumon';
+UPDATE animals SET owner_id = 2 WHERE name = 'Pikachu';
+UPDATE animals SET owner_id = 3 WHERE name = 'Devimon';
+UPDATE animals SET owner_id = 3 WHERE name = 'Plantmon';
+UPDATE animals SET owner_id = 4 WHERE name = 'Charmander';
+UPDATE animals SET owner_id = 4 WHERE name = 'Squirtle';
+UPDATE animals SET owner_id = 4 WHERE name = 'Blossom';
+UPDATE animals SET owner_id = 5 WHERE name = 'Angemon';
```
Comment (BenMukebo) [inline data.sql]: - [OPTIONAL] Great job so far, but I think we can improve this a bit better 🤔

```sql
   UPDATE animals
   SET owner_id = (SELECT id from owners WHERE full_name = 'Jennifer Orwell')
   WHERE name = 'Gabumon' OR name = 'Pikachu';

   UPDATE animals
   SET owner_id = (SELECT id from owners WHERE full_name = 'Bob')
   WHERE name = 'Devimon' OR name = 'Plantmon';

   UPDATE animals
   SET owner_id = (SELECT id from owners WHERE full_name = 'Melody Pond')
   WHERE name = 'Charmander' OR name = 'Squirtle' OR name = 'Blossom';

   UPDATE animals
   SET owner_id = (SELECT id from owners WHERE full_name = 'Dean Winchester')
   WHERE name = 'Angemon'  OR name = 'Boarmon';
```

**Review by BenMukebo — CHANGES_REQUESTED** [review]
Hi @inspecta, 

You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

## To Highlight :clap: 
- Gitflow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and summary. ✔️

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏 💻  
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please Tag me on (@BenMukebo) when you comment so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: data.sql (hunk 1 above)
Comment (inspecta) [reply data.sql:26]: @BenMukebo Thanks for the reminder, I'll include those SQL statements.

File: data.sql (hunk 2 above)
Comment (inspecta) [reply data.sql]: Modified. Thanks.

File: data.sql (hunk 3 above)
Comment (inspecta) [reply data.sql]: :+1:

**Comment by inspecta** [comment]
**Update `species_id` in animals table**

Screenshot from 2022-12-08 01-06-16

** Update `owners` in animals table **
Screenshot from 2022-12-07 22-36-27

**Review by SajjadAhmad14 — APPROVED** [review]
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:

### Hi Derrick, 
## To Highlight 🎉
PR has a title and summary. :heavy_check_mark:
Git flow is followed. :heavy_check_mark:
## Congratulations 🎉 Thank you for implementing all the requirements 🙏
### Please go ahead and merge the branch!

## Happy Coding 👏👏👏
#### Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
#### Please remember to tag me in your question so I can receive the notification.
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**